engine/consensus/matching/engine.go

```diff
@@ -501,19 +501,24 @@ func (e *Engine) checkingSealing() error {
 	sealableResultsSpan := e.tracer.StartSpanFromParent(sealingSpan, trace.CONMatchCheckSealingSealableResults)
 
 	// get all results that have collected enough approvals on a per-chunk basis
-	sealableResults, err := e.sealableResults()
+	sealableResults, nextUnsealeds, err := e.sealableResults()
 	if err != nil {
 		return fmt.Errorf("could not get sealable execution results: %w", err)
 	}
+
+	lg := e.log.With().
+		Int("sealable_results_count", len(sealableResults)).
+		Str("next_unsealed_results", fmt.Sprintf("%v", nextUnsealeds)).
+		Logger()
+
 	if len(sealableResults) == 0 {
-		// skip if no results can be sealed yet
+		lg.Info().Msg("no results can be sealed yet")
```

Comment:
Even if there is no sealable results, we will still print log, so that information about the next unsealed could be logged